cmd/proxy/main.go

```diff
@@ -23,8 +27,25 @@ func main() {
 	flag.StringVar(&visualizeServerPort, "p", "8081", "Port for server to listen on")
 	flag.StringVar(&proxyServerPort, "p", "8081", "Port for server to listen on")
 
-	ctx := context.Background()
-	if err := server.Run(ctx, proxyServerPort, visualizeServerPort, yamlConfigBytes, certificates); err != nil {
-		log.Fatalln(err)
+	quit := make(chan os.Signal)
+
+	signal.Notify(quit, syscall.SIGINT, syscall.SIGTERM)
+
+	go func() {
+		if err := server.Run(proxyServerPort, visualizeServerPort, yamlConfigBytes, certificates); err != nil {
+			log.Fatalln(err)
+		}
+	}()
+
+	log.Println("Server started on :8080")
```

Comment:
I think it must be modified to the correct ports

Reply:
You mean like this:

![image](https://github.com/user-attachments/assets/9cc6ef46-c5e3-43f8-92f9-81f3bae1d676)